server/src/routes/api/v1/trailReviewRouter.js

```diff
@@ -0,0 +1,34 @@
+import express from "express"
+import objection from "objection"
+const { ValidationError } = objection
+
+import { Review } from "../../../models/index.js"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
+const trailReviewRouter = new express.Router({ mergeParams: true })
```

Comment:
To make use of the merge params, we should query to find the trail that the incoming review should belong to and ensure it exists in the database before inserting the review

```js
//depending on the wildcard used in the parent router
const {trailId} = req.params
```

Reply:
Looks like you are using `id`

Reply:
This one's also a little confusing. I'm not sure where we should add `const {trailId} = req.params`

Reply:
Within the `trailReviewRouter.post()` endpoint since that's where we will need to access the `id` used to get the `trail` that the about to be added review should belong to